### ISSUE TITLE
Yaml multiple config on same class but different method

### ISSUE THREAD

**Issue by Jerome1337** [schmittjoh/JMSSerializerBundle#504]
I'm using `JMSSerializerBundle` with `FOSRestBundle`.

I use the `yml` reference to serialize datas on a given class but I have to use another config for a method inside this class but I can't create `yml` config file for a method.

    // This is the class config
    PowerDNSBundle\Entity\PowerDNSDomain:
    exclusion_policy: ALL
    properties:
        id:
            expose: true
            groups: [list, details]
        name:
            expose: true
            groups: [list, details]
        ownerId:
            expose: true
            groups: [list, details]
        createDate:
            expose: true
            groups: [list, details]

    // This is th config I'm trying to use on my method
    PowerDNSBundle\Entity\PowerDNSDomain:
        exclusion_policy: NONE
            properties:
            server:
                exclude: true

This is possible to do this with the YAML reference ?

Thank

**Comment by tuxado** [comment]
i am working on the same config perhabse you have probleme with  exclusion_policy: NONE that overwrite exclusion_policy: ALL

I have a question for you relating the group configuration groups: [list, details] is this configuration work for you? because i use the same list, details and nothing work :(

**Comment by stof** [comment]
The issue is that you have a duplicate key ``PowerDNSBundle\Entity\PowerDNSDomain`` in your YAML file, which does not work (same than duplicate keys in PHP arrays don't work).

**Comment by Jerome1337** [comment]
@tuxado Yes this is working well in my project, I just use add a group to my serialization context with something like this
```
$view->getSerializationContext()->setGroups('list');
```
